src/QsCompiler/LlvmBindings/Values/GlobalIndirectSymbol.cs

```diff
@@ -17,7 +17,6 @@ public class GlobalIndirectSymbol
         public Constant IndirectSymbol
         {
             get => this.Operands.GetOperand<Constant>(0)!;
-            set => this.Operands[0] = value;
```

Comment:
Similar question here, just wondering what the removal was motivated by.

Reply:
Fixed in https://github.com/microsoft/qsharp-compiler/pull/1026/commits/7010cfd901ca34ffc8789dde97143ee9c195bb3a .